specification/liftrastronomer/resource-manager/Astronomer.Astro/preview/2023-08-01-preview/astronomer.json

```diff
@@ -569,7 +569,6 @@
         }
       },
       "required": [
-        "subscriptionId",
```

Comment:
Why is this marked as optional? I am assuming this would be required for all the resources properties.